packages/terra-list/src/ListSectionHeader.jsx

```diff
@@ -107,6 +114,9 @@ const ListSectionHeader = ({
     attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
     attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
   }
+  if (role === 'option' || role === 'listbox') {
```

Comment:
'listbox' isn't a invalid role. Likely here we want to just do presence check, and if a role exists, override the role value.

We'll also want to add the isDisabled behavior/props to this as well.

Reply:
Added